### PR TITLE
Update release notes draft to version v1.32.0-alpha.1

### PR DESCRIPTION
**Pull request by rayandas**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.32.0-alpha.1

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.32/release-notes/maps/pr-122890-map.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+pr: 122890
+releasenote:
+  text: Fixed the bug where spec.terminationGracePeriodSeconds of the pod will always
```
Comment (satyampsoni) [inline releases/release-1.32/release-notes/maps/pr-122890-map.yaml:3]:
```suggestion
  text: Fixed the bug where `spec.terminationGracePeriodSeconds` of the pod will always
```

File: releases/release-1.32/release-notes/maps/pr-127012-map.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+pr: 127012
+releasenote:
+  text: Send bookmark right now after sending all items in watchCache store
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127012-map.yaml:3]:
```suggestion
  text: Sent bookmark right now after sending all items in watchCache store
```

File: releases/release-1.32/release-notes/maps/pr-122890-map.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+pr: 122890
+releasenote:
+  text: Fixed the bug where `spec.terminationGracePeriodSeconds` of the pod will always
+    be overwritten by the MaxPodGracePeriodSeconds of the soft eviction, you can enable
+    the `AllowOverwriteTerminationGracePeriodSeconds` feature gate, which will restore
+    the previous behavior.  If you do need to set this, please file an issue with
+    the Kubernetes project to help contributors understand why you need it.
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-122890-map.yaml:7]:
```suggestion
    the Kubernetes project to help contributors understand why you needed it.
```

File: releases/release-1.32/release-notes/maps/pr-124003-map.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+pr: 124003
+releasenote:
+  text: 'Kube-scheduler removed the following plugins: - AzureDiskLimits - CinderLimits
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-124003-map.yaml:3]:
```suggestion
  text: 'Kube-scheduler removed `AzureDiskLimits` and `CinderLimits` plugin
```

File: releases/release-1.32/release-notes/maps/pr-124003-map.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    - EBSLimits - GCEPDLimits Because the corresponding CSI driver reports how many
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-124003-map.yaml:4]:
```suggestion
    `EBSLimits` and `GCEPDLimits` plugin. Given the corresponding CSI driver reports how many
```

File: releases/release-1.32/release-notes/maps/pr-124003-map.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+    volumes a node can handle in NodeGetInfoResponse, the kubelet stores this limit
+    in CSINode and the scheduler then knows the driver''s limit on the node. Remove
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-124003-map.yaml:6]:
```suggestion
    in CSINode and the scheduler then knows the limit of the driver on the node. Removed
```

File: releases/release-1.32/release-notes/maps/pr-124703-map.yaml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+pr: 124703
+releasenote:
+  text: Implemented a queueing hint for PersistentVolumeClaim/Add event in CSILimit
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-124703-map.yaml:3]:
```suggestion
  text: Implemented a queueing hint for PersistentVolumeClaim/Add event in `CSILimit`
```

File: releases/release-1.32/release-notes/maps/pr-127598-map.yaml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+pr: 127598
+releasenote:
+  text: Fixed a bug with dual stack clusters using the beta feature MultiCIDRServiceAllocator
+    can not create dual stack Services or Services with IPs on the secondary range.
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127598-map.yaml:4]:
```suggestion
    which could not create dual stack Services or Services with IPs in the secondary range.
```

File: releases/release-1.32/release-notes/maps/pr-127598-map.yaml [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+    User that want to use this feature in 1.30 with dual stack clusters can workaround
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127598-map.yaml:5]:
```suggestion
    Users who wanted to use this feature in version 1.30 with dual stack clusters could work around
```

File: releases/release-1.32/release-notes/maps/pr-125398-map.yaml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+pr: 125398
+releasenote:
+  text: Fixed a bug when the hostname label of a node does not match the node name,
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-125398-map.yaml:3]:
```suggestion
  text: Fixed a bug that occurred when the hostname label of a node did not match the node name,
```

File: releases/release-1.32/release-notes/maps/pr-125634-map.yaml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+pr: 125634
+releasenote:
+  text: X.509 client certificate authentication to kube-apiserver now produces credential
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-125634-map.yaml:3]:
```suggestion
  text: X.509 client certificate authentication to the kube-apiserver now produces credential
```

File: releases/release-1.32/release-notes/maps/pr-125634-map.yaml [hunk 12]
```diff
@@ -0,0 +1,5 @@
...
+    IDs (derived from the certificate's signature) for use by audit logging.
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-125634-map.yaml:4]:
```suggestion
    IDs (derived from the certificate's signature) , for use in audit logging.
```

File: releases/release-1.32/release-notes/maps/pr-125796-map.yaml [hunk 13]
```diff
@@ -0,0 +1,11 @@
+pr: 125796
+releasenote:
+  text: 'Fixed a bug in the garbage collector controller which could block indefinitely
+    on a cache sync failure. This fix allows the garbage collector to eventually continue
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-125796-map.yaml:4]:
```suggestion
    due to a cache sync failure. This fix allows the garbage collector to eventually continue
```

File: releases/release-1.32/release-notes/maps/pr-126359-map.yaml [hunk 14]
```diff
@@ -0,0 +1,5 @@
+pr: 126359
+releasenote:
+  text: Fixed CEL estimated cost of expressions that perform equality checks of IPs,
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126359-map.yaml:3]:
```suggestion
  text: Fixed the estimated cost in CEL for expressions that perform equality checks on IPs,
```

File: releases/release-1.32/release-notes/maps/pr-126512-map.yaml [hunk 15]
```diff
@@ -0,0 +1,6 @@
+pr: 126512
+releasenote:
+  text: The transformation_operations_total metric had an additional resource label
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126512-map.yaml:3]:
```suggestion
  text:  included an additional resource labeltransformati in on_operations_total metric
```

File: releases/release-1.32/release-notes/maps/pr-126706-map.yaml [hunk 16]
```diff
@@ -0,0 +1,5 @@
+pr: 126706
+releasenote:
+  text: Fixed kubectl doesn't print image volume when kubectl describe a pod with
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126706-map.yaml:3]:
```suggestion
  text: Fixed an issue where kubectl doesn't print image volume when kubectl describe a pod with
```

File: releases/release-1.32/release-notes/maps/pr-126745-map.yaml [hunk 17]
```diff
@@ -0,0 +1,5 @@
+pr: 126745
+releasenote:
+  text: Improved PVC Protection Controller's scalability by batch-processing PVCs
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126745-map.yaml:3]:
```suggestion
  text:  improved the scalability of the PVC Protection Controller by batch-processing PVCs
```

File: releases/release-1.32/release-notes/maps/pr-126745-map.yaml [hunk 18]
```diff
@@ -0,0 +1,5 @@
...
+    by namespace with lazy live pod listing.
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126745-map.yaml:4]:
```suggestion
   by namespace and implementing lazy live pod listing.
```

File: releases/release-1.32/release-notes/maps/pr-125796-map.yaml [hunk 19]
```diff
@@ -0,0 +1,11 @@
...
+    garbage collecting other resources if a given resource cannot be listed or watched.
+    Any objects in the unsynced resource type with owner references with `blockOwnerDeletion:
+    true` will not be known to the garbage collector. Use of `blockOwnerDeletion`
+    has always been best-effort and racy on startup and object creation, with this
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-125796-map.yaml:8]:
```suggestion
    has always been best-effort and racy on startup and object creation. With this
```

File: releases/release-1.32/release-notes/maps/pr-126532-map.yaml [hunk 20]
```diff
@@ -0,0 +1,5 @@
+pr: 126532
+releasenote:
+  text: Fixed race condition in kube-proxy initialization that could blackhole UDP
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126532-map.yaml:3]:
```suggestion
  text:  Fixed a race condition in the kube-proxy initialization that could cause UDP
```

File: releases/release-1.32/release-notes/maps/pr-126773-map.yaml [hunk 21]
```diff
@@ -0,0 +1,4 @@
+pr: 126773
+releasenote:
+  text: Fixed a warning message about the gce in-tree cloud provider state
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126773-map.yaml:3]:
```suggestion
  text: Fixed a warning message about the gce in-tree cloud provider state.
```

File: releases/release-1.32/release-notes/maps/pr-126851-map.yaml [hunk 22]
```diff
@@ -0,0 +1,4 @@
+pr: 126851
+releasenote:
+  text: Updated github.com/coredns/corefile-migration to v1.0.24
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126851-map.yaml:3]:
```suggestion
  text: Upgraded github.com/coredns/corefile-migration to v1.0.24.
```

File: releases/release-1.32/release-notes/maps/pr-126962-map.yaml [hunk 23]
```diff
@@ -0,0 +1,4 @@
+pr: 126962
+releasenote:
+  text: Fixed a possible memory leak for QueueingHint (alpha feature)
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126962-map.yaml:3]:
```suggestion
  text: Fixed a possible memory leak in the QueueingHint (alpha feature).
```

File: releases/release-1.32/release-notes/maps/pr-126997-map.yaml [hunk 24]
```diff
@@ -0,0 +1,5 @@
+pr: 126997
+releasenote:
+  text: Optimized code, filter podUID is empty string when call this `getPodAndContainerForDevice`
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-126997-map.yaml:3]:
```suggestion
  text: Optimized the code by filtering out empty strings for podUID when calling the`getPodAndContainerForDevice`
```

File: releases/release-1.32/release-notes/maps/pr-127058-map.yaml [hunk 25]
```diff
@@ -0,0 +1,4 @@
+pr: 127058
+releasenote:
+  text: Removed GAed feature gates ServerSideApply/ServerSideFieldValidation
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127058-map.yaml:3]:
```suggestion
  text: Removed the GAed feature gates for `ServerSideApply` and `ServerSideFieldValidation`.
```

File: releases/release-1.32/release-notes/maps/pr-127083-map.yaml [hunk 26]
```diff
@@ -0,0 +1,5 @@
+pr: 127083
+releasenote:
+  text: Implemented new cluster events UpdatePodSchedulingGatesEliminated and UpdatePodTolerations
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127083-map.yaml:3]:
```suggestion
  text: implemented new cluster events `UpdatePodSchedulingGatesEliminated` and `UpdatePodTolerations`
```

File: releases/release-1.32/release-notes/maps/pr-127417-map.yaml [hunk 27]
```diff
@@ -0,0 +1,5 @@
+pr: 127417
+releasenote:
+  text: Fixed a bug on the endpoints controller that did not reconcile the Endpoint
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127417-map.yaml:3]:
```suggestion
  text:  Fixed a bug in the endpoints controller that failed to reconcile the Endpoint
```

File: releases/release-1.32/release-notes/maps/pr-127551-map.yaml [hunk 28]
```diff
@@ -0,0 +1,6 @@
+pr: 127551
+releasenote:
+  text: Fixed the wrong hierarchical structure for the child span and the parent span
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127551-map.yaml:3]:
```suggestion
  text: Fixed the wrong hierarchical structure for both the child span and the parent span
```

File: releases/release-1.32/release-notes/maps/pr-127473-map.yaml [hunk 29]
```diff
@@ -0,0 +1,5 @@
+pr: 127473
+releasenote:
+  text: Improved Node QueueHint in the NodeResource Fit plugin by ignoring unrelated
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127473-map.yaml:3]:
```suggestion
  text: Improved `Node QueueHint` in the `NodeResource Fit` plugin by ignoring unrelated
```

File: releases/release-1.32/release-notes/maps/pr-127421-map.yaml [hunk 30]
```diff
@@ -0,0 +1,5 @@
+pr: 127421
+releasenote:
+  text: Fixed the ability to set the `resolvConf` option in drop-in kubelet configuration
+    files, validates that drop-in kubelet configuration files are in a supported version.
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127421-map.yaml:4]:
```suggestion
    files, which validates that drop-in kubelet configuration files are in a supported version.
```

File: releases/release-1.32/release-notes/maps/pr-127427-map.yaml [hunk 31]
```diff
@@ -0,0 +1,6 @@
+pr: 127427
+releasenote:
+  text: Fixed the bug in NodeUnschedulable that only happens with QHint enabled, which
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127427-map.yaml:3]:
```suggestion
  text: Fixed the bug in `NodeUnschedulable` that only happens with QHint enabled, which
```

File: releases/release-1.32/release-notes/maps/pr-127444-map.yaml [hunk 32]
```diff
@@ -0,0 +1,5 @@
+pr: 127444
+releasenote:
+  text: Improved Node QueueHint in the NodeAffinty plugin by ignoring unrelated changes
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127444-map.yaml:3]:
```suggestion
  text: Improved `Node QueueHint` in the `NodeAffinty` plugin by ignoring unrelated changes
```

File: releases/release-1.32/release-notes/maps/pr-127016-map.yaml [hunk 33]
```diff
@@ -0,0 +1,5 @@
+pr: 127016
+releasenote:
+  text: Fixed a potential memory leak in QueueingHint (alpha feature)
```
Comment (cloudmelon) [inline releases/release-1.32/release-notes/maps/pr-127016-map.yaml:3]:
```suggestion
  text: Fixed a potential memory leak in QueueingHint (alpha feature).
```

File: releases/release-1.32/release-notes/maps/pr-127598-map.yaml [hunk 34]
```diff
@@ -0,0 +1,7 @@
+pr: 127598
+releasenote:
+  text: Fixed a bug with dual stack clusters using the beta feature MultiCIDRServiceAllocator
+    which could not create dual stack Services or Services with IPs in the secondary range.
+    Users who wanted to use this feature in version 1.30 with dual stack clusters could work around
+    the issue by setting the feature gate DisableAllocatorDualWrite to true
```
Comment (npolshakova) [inline releases/release-1.32/release-notes/maps/pr-127598-map.yaml:6]:
```suggestion
    the issue by setting the feature gate DisableAllocatorDualWrite to true.
```

File: releases/release-1.32/release-notes/maps/pr-127012-map.yaml [hunk 35]
```diff
@@ -0,0 +1,4 @@
+pr: 127012
+releasenote:
+  text: Sent bookmark right now after sending all items in watchCache store.
```
Comment (npolshakova) [inline releases/release-1.32/release-notes/maps/pr-127012-map.yaml:3]: Maybe reword based on the longer description? https://github.com/kubernetes/kubernetes/pull/127012 

```suggestion
  text: Bookmark events are now sent immediately after all items in the watchCache store have been processed, improving consistency in client behavior.
```

File: releases/release-1.32/release-notes/maps/pr-126706-map.yaml [hunk 36]
```diff
@@ -0,0 +1,5 @@
+pr: 126706
+releasenote:
+  text: Fixed an issue where kubectl doesn't print image volume when kubectl describe a pod with
+    that volume
```
Comment (npolshakova) [inline releases/release-1.32/release-notes/maps/pr-126706-map.yaml:4]:
```suggestion
    that volume.
```

**Review by npolshakova — COMMENTED** [review]
Looking good! Have you tried running the `krel release-notes validate` command to check if all the edited yaml is valid?

File: releases/release-1.32/release-notes/release-notes-draft.md [hunk 37]
```diff
@@ -0,0 +1,169 @@
...
+- When SchedulerQueueingHints is enabled, clear events cached in the scheduling queue as soon as possible so that the scheduler consumes less memory. ([#120586](https://github.com/kubernetes/kubernetes/pull/120586), [@sanposhiho](https://github.com/sanposhiho)) [SIG Scheduling]
+
+### Documentation
+
+- Clarified the kube-controller-manager documentation for --allocate-node-cidrs, --cluster-cidr, and --service-cluster-ip-range flags to accurately reflect their dependencies and usage conditions. ([#126784](https://github.com/kubernetes/kubernetes/pull/126784), [@eminwux](https://github.com/eminwux)) [SIG API Machinery, Cloud Provider and Docs]
+- Documented the `--for=create` option to `kubectl wait` ([#127327](https://github.com/kubernetes/kubernetes/pull/127327), [@ryanwinter](https://github.com/ryanwinter)) [SIG CLI]
+
+### Failing Test
+
+- Kubelet Plugins are now re-registered properly  on Windows if the re-registration period is < 15ms. ([#114136](https://github.com/kubernetes/kubernetes/pull/114136), [@claudiubelu](https://github.com/claudiubelu)) [SIG Node, Storage, Testing and Windows]
```
Comment (npolshakova) [inline releases/release-1.32/release-notes/release-notes-draft.md:81]: Extra space? 

```suggestion
- Kubelet Plugins are now re-registered properly on Windows if the re-registration period is < 15ms. ([#114136](https://github.com/kubernetes/kubernetes/pull/114136), [@claudiubelu](https://github.com/claudiubelu)) [SIG Node, Storage, Testing and Windows]
```

**Comment by fsmunoz** [comment]
Looking good, thanks!

/lgtm
/approve

**Comment by satyampsoni** [comment]
Thanks @fsmunoz
